### PR TITLE
Shopping cart 1

### PR DESCRIPTION
**Pull request by mimilettd**
#6 

Description of Changes Made: To implement the cart, I created a `CartsController`, `Cart` model, and `cart#index` view. I added a cart link to the `item#index` view. I also made the cart persistent by adding it in the `application.html.erb` file. Users can now add an item, see a flash message confirming that the item was placed in the cart, see the **flash** message QUANTITY increment by one, see the **cart** QUANTITY increment, and can click the cart to view all the items they've added.

Anticipated Code Effects: I refactored the `item#index` view so that might cause a possible conflict. I also moved the yield down in the `application.html.erb`.
(Would this merge affect any other piece of our code?) 

Reviewer: @rickyamparo should be the primary reviewer but @samuelssnider and @JunePaloma should still take a look at this.

### CODE REVIEW

**Comment by mimilettd** [comment]
@rickyamparo - the comments were actually removed before the pull request. I'm surprised it's still persisting but regardless, I removed it directly using the online text editor. To answer your second question, I had to hop onto a new branch (which resolved the problem). Everything should be good now.

**Comment by rickyamparo** [comment]
@mimilettd  Cool Deal! Yeah it seems like there's some discrepancies between git and GitHub. Everything else looks really good and I'm read to merge.

File: spec/helpers.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Helpers
+  def create_items
+    @category = Category.create(title: "Cats")
+    @item = @category.items.create(title: "Black Cat Onesie",
+                        description: "The best cat onesie you'll ever find",
+                        price: 19.99,
+                        image: "https://images-na.ssl-images-amazon.com/images/I/41CFHIae7TL._SL500_.jpg")
+  end
```
Comment (samuelssnider) [inline spec/helpers.rb]: Like this method, might need to refactor my tests with something similar

Comment (mimilettd) [reply spec/helpers.rb]: @samuelssnider Yes! I think building on this `helpers.rb` will help us keep our code DRY. 👍 Let me know if you have any questions when you start refactoring. Remember to make your variables an instance variable so you can call them in your own test file.

**Pull request merged**